delft3dworker/models.py

```diff
@@ -931,11 +934,9 @@ def remove_workflow(self):
             args=(self.name,),
             expires=settings.TASK_EXPIRE_TIME
         )
-
-        self.task_starttime = now()
```

Comment:
We should not remove task_starttime, as it's used still. (see models.py, update_task_result function).